windows-utils/replace-symlinks.js

```diff
@@ -0,0 +1,94 @@
+const shell = require("shelljs");
+
+async function getSymlinks(dirPath) {
+    let symlinks = [];
+    const path = dirPath
+        .replaceAll("/", "\\")
+        .replaceAll("../", "")
+        .split("\\")
+        .slice(2)
+        .join("\\");
+    const cwd = process.cwd();
+    const changeDir = cwd + path;
+    const fileList = await shellExec("git ls-files -s");
+
+    if (fileList.stdout.includes("120000")) {
+        const output = fileList.stdout.split("\n");
+        symlinks.push(...extractSymlinks(output, changeDir));
+    }
+    shell.cd(cwd);
+    return symlinks;
+}
+const shellExec = (cmd) =>
+    new Promise((resolve) => {
+        shell.exec(cmd, (code, stdout, stderr) => {
+            resolve({ code, stdout, stderr });
+        });
+    });
+
+function extractSymlinks(output, cwd) {
+    const extractedSymlinks = [];
+    for (let i = 0; i < output.length; i++) {
+        const parsedElement = output[i].split(" ");
```

Comment:
Also a bit old school, you could use destructuring here I guess (and avoid splitting up the remainder further down below):

    const [mode, id, _, relDir] = output[i].split(/\s/);